torch/csrc/Generator.cpp

```diff
@@ -30,7 +30,7 @@ using namespace torch;
 
 PyObject* THPGeneratorClass = nullptr;
 
-PyObject* THPGenerator_initDefaultGenerator(const at::Generator& cdata) {
+PyObject* THPGenerator_initDefaultGenerator(at::Generator cdata) {
```

Comment:
so at::Generator is trivially copyable now?

Reply:
It's moved in the body